### DISCUSSION TITLE
Cant create subscription batches

### DISCUSSION THREAD

**Discussion by universowebstore** [sensson/whmcs-incasso#10] [Q&A]
https://cdn.discordapp.com/attachments/1197497555957850182/1236977005900075030/image.png?ex=663aa0dc&is=66394f5c&hm=452d7fb1df054583eae71228c393f83407401f2da9f1ce0bb19e6704d7f57d8a&

**Comment by ju5t** [comment]
Sorry but I cannot help you with this. It says there are no invoices to process. That's not an error. If you're not asking any questions it's impossible to help.

**Comment by universowebstore** [comment]
my boss send me that and said i cant create that, i used to do it before but now it said there isnt invoices to do that,

**Comment by ju5t** [comment]
But this is not an issue? It says there are no invoices marked for direct debit that are unpaid. 

If that's not right you need to check the settings and make sure either the invoices are set to the correct payment method or that the client's payment method is configured for direct debit.

This is 100% not an issue in the application.

**Comment by universowebstore** [comment]
Hello, there are unpaid invoices, and i cant create sub batches.
![incasso](https://github.com/sensson/whmcs-incasso/assets/168188075/63ff4fd4-6e88-4d62-882e-58509be459d1)
